frontend/pages/admin/manage/users/index.vue

```diff
@@ -2,9 +2,21 @@
   <v-container fluid>
     <BaseDialog v-model="deleteDialog" :title="$t('general.confirm')" color="error" @confirm="deleteUser(deleteTarget)">
       <template #activator> </template>
-      <v-card-text>
+
+      <v-card-text
+        v-if="isUserOwnAccount === true">
+        <v-alert
+        text
+        outlined
+        color="deep-orange"
+        >⚠️ Please note that you are trying to delete your own admin account!
```

Comment:
Should put this inside a language file so it's more easily translatable

Reply:
Thanks for the feedback! I put the text inside the language file.